nixos/modules/system/boot/systemd.nix

```diff
@@ -804,10 +804,10 @@ in
         [Manager]
         ${optionalString config.systemd.enableCgroupAccounting ''
           DefaultCPUAccounting=yes
+          DefaultBlockIOAccounting=yes
           DefaultIOAccounting=yes
           DefaultBlockIOAccounting=yes
-          DefaultMemoryAccounting=yes
```

Comment:
Why MemoryAccounting have to go (and Tasks for that matter)?

Reply:
They're enabled by default - see the commit message doing this change.